cysync/src/store/hooks/flows/useDeviceUpgrade.ts

```diff
@@ -568,8 +571,6 @@ export const useDeviceUpgrade: UseDeviceUpgrade = (isInitial?: boolean) => {
         setErrorResolutionState(
           DeviceUpgradeErrorResolutionState.DEVICE_AUTH_REQUIRED
         );
-        const cyError = new CyError(CysyncError.DEVICE_AUTH_FAILED);
-        setErrorObj(handleErrors(errorObj, cyError, flowName));
```

Comment:
We're not setting error here?

Reply:
Not necessary, on device auth failure. The errors are set by `useDeviceAuth` hook itself. So if there's any auth error, `useDeviceUpgrade` is expected to just forward it.